elixir/apps/web/lib/web/live/policies/index.ex

```diff
@@ -1,5 +1,13 @@
 defmodule Web.Policies.Index do
   use Web, :live_view
+  alias Domain.Policies
+
+  def mount(_params, _session, socket) do
+    {:ok, policies} =
```

Comment:
This one can return an error too (unauthorized) so we also need to raise 404 error for that one.

Reply:
This has been updated.